lib/cdo/shared_constants.rb

```diff
@@ -642,6 +642,33 @@ module SharedConstants
     }
   ).freeze
 
+  VOICES = {
```

Comment:
I see there was a TODO where these constants originally were. Is there a reason why these are shared with Dashboard? If they're defined separately in Dashboard currently, then we should switch the references from wherever they are in Dashboard to `shared_constants`.